### ISSUE TITLE
Can't use otlp/jaeger as exporter

### ISSUE THREAD

**Issue by sterziev88** [open-telemetry/opentelemetry-helm-charts#1073]
Hello,

I am trying to collect and send traces via opentelemetry to my jaeger.
At the moment I use jaeger UI to see my traces and my applications use zipkin to send that traces.
I want to use opentelemetry and the is what I have configured until now:

>  config:
      exporters:
        debug:
          verbosity: detailed
          sampling_initial: 5
          sampling_thereafter: 200
        # Will be removed in a future release.
        # Use the debug exporter instead.
        logging: {}
        otlp/jaeger:
          endpoint: "jaeger-collector.jaeger.svc.cluster.local:4317"
          tls:
            insecure: true
      extensions:
        # The health_check extension is mandatory for this chart.
        # Without the health_check extension the collector will fail the readiness and liveliness probes.
        # The health_check extension can be modified, but should never be removed.
        health_check:
          endpoint: ${env:MY_POD_IP}:13133
        memory_ballast: {}
      processors:
        batch: {}
        # If set to null, will be overridden with values based on k8s resource limits
        memory_limiter: null
      receivers:
        jaeger:
          protocols:
            grpc:
              endpoint: ${env:MY_POD_IP}:14250
            thrift_http:
              endpoint: ${env:MY_POD_IP}:14268
            thrift_compact:
              endpoint: ${env:MY_POD_IP}:6831
        otlp:
          protocols:
            grpc:
              endpoint: ${env:MY_POD_IP}:4317
            http:
              endpoint: ${env:MY_POD_IP}:4318
        prometheus:
          config:
            scrape_configs:
              - job_name: opentelemetry-collector
                scrape_interval: 10s
                static_configs:
                  - targets:
                      - ${env:MY_POD_IP}:8888
        zipkin:
          endpoint: ${env:MY_POD_IP}:9411
      service:
        telemetry:
          metrics:
            address: ${env:MY_POD_IP}:8888
        extensions:
          - health_check
          - memory_ballast
        pipelines:
          traces:
            exporters:
              - debug
              - otlp/jaeger
            processors:
              - memory_limiter
              - batch
            receivers:
              - otlp
              - jaeger

but I receive an error:
2024-03-13T13:08:58.755Z warn zapgrpc/zapgrpc.go:195 [core] [Channel #1 SubChannel #2] grpc: addrConn.createTransport failed to connect to {Addr: "jaeger-collector.jaeger.svc.cluster.local:4317", ServerName: "jaeger-collector.jaeger.svc.cluster.local:4317", }. Err: connection error: desc = "transport: Error while dialing: dial tcp 172.20.136.48:4317: connect: connection refused" {"grpc_log": true}

If add this for endpoint:
         tls:
            insecure: true
I receive the same error.

Any suggestions?

**Comment by JaredTan95** [comment]
@sterziev88 Obviously, the jaeger is not connected, you need to check your jaeger address is correct
